superset/config.py

```diff
@@ -799,6 +799,12 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # and render for the email report.
 EMAIL_PAGE_RENDER_WAIT = 30
 
+# Time in seconds before selenium times out after trying to
+# locate an element on the page and wait for that element to
+# load for an alert screenshot.
```

Comment:
Let's add a note in this comment block that this is used for alerting functionality - we may also want to apply the same configuration to report generation if it is not already so.

Reply:
I find it useful to keep those separate while iterating on the solution. E.g. we have email reports deployed in production but alerts is still work in progress. 
+1 to add the comment that those are the nobs for the thumbnail endpoint / alerting [ not 100 % sure which one is used here]

Reply:
That's fine. Let's bring them together at some point though!